src/modules/commander/state_machine_helper.cpp

```diff
@@ -310,11 +310,9 @@ main_state_transition(struct vehicle_status_s *status, main_state_t new_main_sta
 		break;
 
 	case vehicle_status_s::MAIN_STATE_ALTCTL:
-		/* need at minimum altitude estimate */
-		/* TODO: add this for fixedwing as well */
-		if (!status->is_rotary_wing ||
```

Comment:
The solution with circuit breaker is inconvenient for people who fly with GPS and perform tests indoor as well (without vicon). It adds up a parameter that has to be checked and forgetting about it could potentially lead to the same corner case.